data/json/items/gun/monster_gun.json

```
@@ -1,110 +1,85 @@
-[{
-		"id": "barb_launcher",
-		"type": "GUN",
-		"symbol": "%",
-		"color": "red",
-		"name": {
-			"str": "barb launching organ"
-		},
-		"description": "A mutated organ capable of launching bony barbs at great speed.",
-		"material": ["hflesh"],
-		"flags": [
-			"PRIMITIVE_RANGED_WEAPON",
-			"NEVER_JAMS",
-			"NONCONDUCTIVE",
-			"NO_REPAIR",
-			"WATERPROOF_GUN",
-			"NO_SALVAGE",
-			"NO_UNLOAD",
-			"RELOAD_AND_SHOOT"
-		],
-		"skill": "pistol",
-		"ammo": ["barb"],
-		"clip_size": 1,
-		"weight": "540 g",
-		"volume": "750ml",
-		"bashing": 2,
-		"to_hit": 1,
-		"reload_noise_volume": 2,
-		"loudness": 2,
-		"range": 12,
-		"dispersion": 100,
-		"durability": 8,
-		"pocket_data": [{
-			"pocket_type": "MAGAZINE",
-			"rigid": true,
-			"ammo_restriction": {
-				"barb": 1
-			}
-		}]
-	},
-	{
-		"id": "emp_frond",
-		"type": "GUN",
-		"symbol": "%",
-		"color": "red",
-		"name": {
-			"str": "electric alien frond"
-		},
-		"description": "Electricity unnaturally arcs from the tips of this alien frond.",
-		"material": ["hflesh"],
-		"flags": [
-			"PRIMITIVE_RANGED_WEAPON",
-			"NEVER_JAMS",
-			"NONCONDUCTIVE",
-			"NO_REPAIR",
-			"WATERPROOF_GUN",
-			"NO_SALVAGE",
-			"NO_UNLOAD",
-			"NO_AMMO"
-		],
-		"skill": "pistol",
-		"ammo_effects": ["EMP", "LIGHTNING"],
-		"ranged_damage": {
-			"damage_type": "electric",
-			"amount": 1
-		},
-		"weight": "540 g",
-		"volume": "750ml",
-		"bashing": 2,
-		"to_hit": 1,
-		"reload_noise_volume": 2,
-		"loudness": 2,
-		"range": 12,
-		"dispersion": 150,
-		"durability": 8
-	},
-	{
-		"id": "feral_human_thrown_rock",
-		"type": "GUN",
-		"symbol": "%",
-		"color": "red",
-		"name": {
-			"str": "hurled rubble"
-		},
-		"description": "Stone at the ready to crush that which isn't part of the blob.",
-		"material": ["stone"],
-		"flags": [
-			"PRIMITIVE_RANGED_WEAPON",
-			"NEVER_JAMS",
-			"NONCONDUCTIVE",
-			"NO_REPAIR",
-			"WATERPROOF_GUN",
-			"NO_SALVAGE",
-			"NO_UNLOAD"
-		],
-		"skill": "throw",
-		"ranged_damage": {
-			"damage_type": "bash",
-			"amount": 1
-		},
-		"weight": "540 g",
-		"volume": "750ml",
-		"bashing": 2,
-		"to_hit": 1,
-		"loudness": 2,
-		"range": 6,
-		"dispersion": 150,
-		"durability": 8
-	}
+[
+  {
+    "id": "barb_launcher",
+    "type": "GUN",
+    "symbol": "%",
+    "color": "red",
+    "name": { "str": "barb launching organ" },
+    "description": "A mutated organ capable of launching bony barbs at great speed.",
+    "material": [ "hflesh" ],
+    "flags": [
+      "PRIMITIVE_RANGED_WEAPON",
+      "NEVER_JAMS",
+      "NONCONDUCTIVE",
+      "NO_REPAIR",
+      "WATERPROOF_GUN",
+      "NO_SALVAGE",
+      "NO_UNLOAD",
+      "RELOAD_AND_SHOOT"
+    ],
+    "skill": "pistol",
+    "ammo": [ "barb" ],
+    "clip_size": 1,
+    "weight": "540 g",
+    "volume": "750ml",
```

```suggestion
    "volume": "750 ml",
```
Applies to other "volume" instances in the file as well.

---

It's done!
I'm gonna close this now.